tripal/src/TripalField/TripalFieldItemBase.php

```diff
@@ -176,19 +179,20 @@ public function fieldSettingsForm(array $form, FormStateInterface $form_state) {
     ];
 
     $default_vocabulary_term = '';
-    $vocabulary_term = $form_state->getValue(['settings', 'field_term_fs', 'vocabulary_term']);
+    // For Drupal ~10.2 our values are now in the subform
```

Comment:
Perhaps we should getFormStateSettings() here as well now that we have that function?

Reply:
The `getFormStateSettings()` looks like it works directly with the `settings` element of the subform, but the `vocabulary_term` form element isn't stored in the settings. It's used to help collect the term name and cvterm_id for the term and then in the validate function the setting actually get set, so I think we need to leave this as it is.